cmd/azure-config-credentials-injector/credentials_injector_test.go

```diff
@@ -0,0 +1,150 @@
+package main
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/assert"
+	"io/ioutil"
+	"os"
+	"testing"
```

Comment:
Nit: standard library imports should come first, with a gap, then the additional imports
```suggestion
	"io/ioutil"
	"os"
	"testing"
	
	"github.com/spf13/cobra"
	"github.com/stretchr/testify/assert"
```